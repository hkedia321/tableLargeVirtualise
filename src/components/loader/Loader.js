import React from 'react';
import ClipLoader from "react-spinners/ClipLoader";

function Loader() {
    return (
        <div align="center">
        <ClipLoader
          size={100}
          color={"#123abc"}
        />
        </div>
    )
}

export default Loader;