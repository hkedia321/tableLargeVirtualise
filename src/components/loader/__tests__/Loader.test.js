import React from "react"
import { render, cleanup } from "@testing-library/react"
import renderer from "react-test-renderer"
import Loader from ".."

afterEach(cleanup)

it("renders Loader without crashing", () => {
  const { getByTestId } = render(<Loader />)
  expect(getByTestId("loader"))
})

it("matches snapshot", () => {
  const tree = renderer.create(<Loader />).toJSON()
  expect(tree).toMatchSnapshot()
})
