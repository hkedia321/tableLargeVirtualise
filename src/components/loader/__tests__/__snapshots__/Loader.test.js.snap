// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  align="center"
  className="sc-AxjAm bqAWmr"
  data-testid="loader"
>
  <div
    className="css-m2td1u"
  >
    <div
      className="css-5a4nd"
    />
    <div
      className="css-69adw1"
    />
  </div>
</div>
`;
